### PR TITLE
[PERFSCALE-3525] Add OVN PerfScale AWS Cluster Profile

### PR DESCRIPTION
**Pull request by krishvoor**
Add OVN PerfScale AWS Cluster Profile
Depends on: https://github.com/openshift/ci-tools/pull/4379

### CODE REVIEW

**Comment by krishvoor** [comment]
/assign @smg247 

@smg247 Can you help review this PR, thank you!

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -2258,6 +2258,10 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-azure-qe
     namespace: ci
+  - cluster_groups:
+    - non_app_ci:
```
Comment (jtaleric) [inline core-services/ci-secret-bootstrap/_config.yaml:2262]: i don't think there should be a `:` at the end of this line?

Comment (krishvoor) [reply core-services/ci-secret-bootstrap/_config.yaml]: Good catch, updated the logic!

**Review by krishvoor — COMMENTED** [review]
/lgtm

**Pull request merged**